### PR TITLE
Group Members & Larger Entities in Group Reports #960

### PR DESCRIPTION
**Pull request by johnatawnclementawn**
The Group Members and Larger Entities tabs on the group report are now populated with the people and/or groups that are belong to a group or encapsulate the current group, respectively. A note about this implementation as it stands, only people and groups that are directly related will be shown. That is to say, any person or group with 2 degrees of separation from the current group will not be shown on this report.

### CODE REVIEW

**Comment by aarongundel** [comment]
@johnatawnclementawn can you resolve the merge conflict here?

**Comment by johnatawnclementawn** [comment]
@aarongundel resolved conflicts. Ready for merge.

File: afs/templates/views/components/reports/group.htm [hunk 1]
```diff
@@ -86,14 +86,97 @@ <h1 class="afs-report-toolbar-title"><span class="afs-report-name">{% trans "Gro
...
             <!-- Larger Entities Tab --> 
             <div class="afs-report-page" data-bind="if: activeSection() === 'entities'">
-                <div data-bind="component: {
-                    name: 'views/components/reports/scenes/default',
-                    params: {
-                        data: parthoodData,
-                    }
-                }"></div>
+                <div class="afs-report-section">
+                    <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.largerEntities)}, css: {'fa-angle-double-right': visible.largerEntities(), 'fa-angle-double-up': !visible.largerEntities()}" class="fa toggle"></i> {% trans "Group Members" %}</h2>
```
Comment (aarongundel) [inline afs/templates/views/components/reports/group.htm:138]: The text here should probably be changed to "Larger Entitites"

**Review by aarongundel — CHANGES_REQUESTED** [review]
This looks good and works for me, just one issue with a header.

**Review by aarongundel — APPROVED** [review]
Looks good!

**Pull request merged**